templates/automotive/buyers-guide/_form.html

```diff
@@ -0,0 +1,96 @@
+<div class="row u-sv3">
+  <hr class="p-rule is-dark">
+  <div class="col-6">
+    <h2>Download our buyer's guide</h2>
+  </div>
+  <div class="col-6 u-sv3">
+    <p class="p-heading--five">See if Ubuntu meets your needs and evaluate how it compares to others for your unique use case.</p>
+    <p>Based on our extensive experience in embedded systems, we put together an overview of considerations that will help you select the right OS for your in-vehicle projects. Our buyer's guide will also help you evaluate if Ubuntu is a good fit for you. Download the guide.</p>
+  </div>
+  <div class="row u-sv3">
+    <div class="col-6 col-start-large-7">
+      <h3 class="p-muted-heading">Contact us</h3>
+    </div>
+  </div>
+  <div class="row">
+    <div class="col-3">
+      {{ image (
+        url="https://assets.ubuntu.com/v1/2395a6a4-automotive-os-buyers-guide.png",
+        alt="",
+        width="599",
+        height="838",
+        hi_def=True,
+        loading="auto|lazy"
+        ) | safe
+      }}
+    </div>
+    <div class="col-6 col-start-large-7">
+      <form action="/marketo/submit" method="post" id="mktoForm_5356">
+          <ul class="p-list">
+            <li class="p-list__item">
+              <label for="firstName">First name:</label>
+              <input required id="firstName" name="firstName" maxlength="255" type="text" />
+            </li>
+            <li class="p-list__item">
+              <label for="lastName">Last name:</label>
+              <input required id="lastName" name="lastName" maxlength="255" type="text" />
+            </li>
+            <li class="p-list__item">
+              <label for="email">Email:</label>
+              <input required id="email" name="email" maxlength="255" type="email" pattern="^[^ ]+@[^ ]+\.[a-z]{2,26}$" />
+            </li>
+           <li class="p-list__item">
+              <label for="company">Company:</label>
+              <input required id="company" name="company" maxlength="255" type="text" />
+            </li>
+           <li class="p-list__item">
+              <label for="title">Title:</label>
```

Comment:
```suggestion
              <label for="title">Job title:</label>
```
Not sure if it matters, but we nearly always use 'Job title' here